ui/pages/swaps/prepare-swap-page/index.scss

```diff
@@ -133,10 +133,10 @@
 
   &__from-token-amount {
     border: 0;
-    outline: none;
+    margin-right: -4px;
 
     input {
-      padding-right: 0;
+      padding-right: 4px;
```

Comment:
Negative margin allows for alignment and padding of number to remain the same while input is in focus state